packages/release-trigger/Dockerfile

```diff
@@ -14,9 +14,33 @@
 
 # Use a multi-stage docker build to limit production dependencies.
 
+FROM node:18.20.3-bullseye AS BASE
+
+# Install pyenv dependencies
+RUN apt-get update && \
+    apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev \
+    libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+    xz-utils tk-dev libffi-dev liblzma-dev python3-openssl
+
+USER node
+
+# Install pyenv
+RUN git clone https://github.com/pyenv/pyenv.git ~/.pyenv && \
+    echo 'export PATH="/home/node/.pyenv/bin:$PATH"' >> ~/.profile && \
+    echo 'export PATH="/home/node/.pyenv/shims:$PATH"' >> ~/.profile && \
+    echo 'eval "$(pyenv init -)"' >> ~/.profile && \
+    echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.profile
+
+ENV PATH="/home/node/.pyenv/bin:/home/node/.pyenv/shims:${PATH}"
+
+# Install python
+RUN pyenv install 3.10.13 && \
```

Comment:
Also here, maybe Renovate can be configured to manage this version.

Reply:
Yes, let's track it as a separate issue.

Reply:
#5366 5366